pallets/rc-migrator/src/lib.rs

```diff
@@ -151,17 +158,30 @@ pub enum MigrationStage<AccountId> {
 		next_key: Option<NomPoolsStage<AccountId>>,
 	},
 	NomPoolsMigrationDone,
+	FastUnstakeMigrationInit,
+	FastUnstakeMigrationOngoing {
+		next_key: Option<FastUnstakeStage<AccountId>>,
+	},
+	FastUnstakeMigrationDone,
 	ReferendaMigrationInit,
 	ReferendaMigrationOngoing {
 		last_key: Option<ReferendaStage>,
 	},
 	ReferendaMigrationDone,
+	BagsListMigrationInit,
+	BagsListMigrationOngoing {
+		next_key: Option<BagsListStage<AccountId, BagsListScore>>,
+	},
+	BagsListMigrationDone,
 	MigrationDone,
 }
 
-pub type MigrationStageFor<T> = MigrationStage<<T as frame_system::Config>::AccountId>;
+pub type MigrationStageFor<T> = MigrationStage<
```

Comment:
Do we have some non written convention when we use `For` or `Of` suffix? I wished to keep it consistent (= 
I had a feeling that Gav used `For` for storage items and `Of` for types, but I am not sure.

Reply:
I was wondering about the same thing. Core issue is that there is `AccountIdOf` and `BlockNumberFor`, so two very central types in contradiction 😆 🤦‍♂️   

I dont really mind when we use what, we can use `Of` for types if you want.